.taprc

```diff
@@ -0,0 +1,15 @@
+esm: false
```

Comment:
so you've got eslint config in the package.json and commitlint config in the package.json. why not also the tap config?

Reply:
I know you love that config in the package :). TBH it was just copy/pasted and it's for the reasons you and I have talked before about tap not playing nice with overrides when it's in the package.

Reply:
I'll move it into the package to be consistent.